app/views/pages/home.html.haml

```diff
@@ -0,0 +1,18 @@
+%section.container
+  %h1= t('welcome')
+  %h1= t('app_name')
+  %h2= t('links')
+
+  %p
+    - if signed_in?
+      %li= link_to t('log_out'), destroy_user_session_path, method: :delete, class: "btn btn-primary btn-sm mt-10"
+      %li= link_to t('profile'),users_profile_path(current_user.id) ,method: :get, class: "btn btn-pimary btn-sm mt-10"
```

Comment:
`link_to t('profile'), users_profile_path(current_user.id), method: :get, class: "btn btn-pimary btn-sm mt-10"` (spaces)

Reply:
you didn't fix this